src/styles/main.scss

```diff
@@ -1,3 +1,27 @@
-body {
+body,
+main,
+header,
+nav,
+div,
+p,
+img,
+ul {
   margin: 0;
+  padding: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@import "./blocks/variables.scss";
+@import "./blocks/placeholders.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
```

Comment:
`variables.scss` and `placeholders.scss` are not blocks. Better move them to folder called `utils`.